the gep lgtm, thanks.

I will leave my approval and the "lgtm" for @LiorLieberman . 

Also we need the metadata.yml file to be fixed according to the linter error:

```
 ./geps/gep-4768/metadata.yaml
  9:2       error    wrong indentation: expected 2 but found 1  (indentation)
```

--- a/geps/gep-4768/index.md
+++ b/geps/gep-4768/index.md
@@ -43,11 +43,11 @@ In the current Kubernetes landscape, the "Who, What, Where, and How Long" of net
 
 ### Policy Attachment vs. Inline Configuration
 
-A key area of discussion for this GEP is whether this should be a standalone Policy Attachment (e.g., `TelemetryPolicy`) or inline configuration within `Gateway`, or `HTTPRoute` resources. 
+A key area of discussion for this GEP is whether this should be a standalone Policy Attachment (e.g., `TelemetryPolicy`) or inline configuration within `Gateway` or `HTTPRoute` resources.
 
 This proposal argues that the Policy Attachment model is the most effective approach to meet the stated goals, primarily for two reasons:
 
-1. **Separation of Concerns**: It allows different personas to manage Gateway infrastructure independently from the configuration of telemetry signals.
+1. **Separation of Concerns**: It allows different personas to manage Gateway infrastructure independently from the configuration of telemetry signals. Telemetry is typically configured by platform, observability, or security engineers rather than application developers. This also implies that HTTPRoute is not the ideal resource to target for the initial API implementation.
 2. **Uniformity**: It enables a single policy to be applied uniformly across a set of Gateways, eliminating the need to duplicate complex telemetry configurations across individual resources.
 
 To mitigate the challenge of complex merging semantics, this GEP restricts configuration such that only a single `TelemetryPolicy` can target a specific `Gateway` at any given time. If multiple `TelemetryPolicy` resources target the same object, precedence is determined based on the creation timestamp. This will allow us to start with simple config and iterate based on feedback whether multiple TelemetryPolicies on the same target are needed.

--- a/geps/gep-4768/index.md
+++ b/geps/gep-4768/index.md
@@ -43,14 +43,14 @@ In the current Kubernetes landscape, the "Who, What, Where, and How Long" of net
 
 ### Policy Attachment vs. Inline Configuration
 
-A key area of discussion for this GEP is whether this should be a standalone Policy Attachment (e.g., `TelemetryPolicy`) or inline configuration within the `Gateway` resource. 
+A key area of discussion for this GEP is whether this should be a standalone Policy Attachment (e.g., `TelemetryPolicy`) or inline configuration within `Gateway`, or `HTTPRoute` resources. 
 
 This proposal argues that the Policy Attachment model is the most effective approach to meet the stated goals, primarily for two reasons:
 
 1. **Separation of Concerns**: It allows different personas to manage Gateway infrastructure independently from the configuration of telemetry signals.
 2. **Uniformity**: It enables a single policy to be applied uniformly across a set of Gateways, eliminating the need to duplicate complex telemetry configurations across individual resources.
 
-To mitigate the challenge of complex merging semantics, this GEP restricts configuration such that only a single `TelemetryPolicy` can target a specific `Gateway` at any given time. If multiple `TelemetryPolicy` resources target the same object, precedence is determined based on the creation timestamp.
+To mitigate the challenge of complex merging semantics, this GEP restricts configuration such that only a single `TelemetryPolicy` can target a specific `Gateway` at any given time. If multiple `TelemetryPolicy` resources target the same object, precedence is determined based on the creation timestamp. This will allow us to start with simple config and iterate based on feedback whether multiple TelemetryPolicies on the same target are needed.
 
 ### High-level Considerations:
 

--- a/geps/gep-4768/metadata.yaml
+++ b/geps/gep-4768/metadata.yaml
@@ -6,6 +6,6 @@ status: Provisional
 authors:
   - gkhom
 seeAlso:
- - https://github.com/kubernetes-sigs/kube-agentic-networking/pull/69
- - https://gateway-api.sigs.k8s.io/geps/gep-713/
+  - https://github.com/kubernetes-sigs/kube-agentic-networking/pull/69
+  - https://gateway-api.sigs.k8s.io/geps/gep-713/
 

